extensions/amp-story/1.0/page-advancement.js

```diff
@@ -430,7 +430,10 @@ class ManualAdvancement extends AdvancementConfig {
       dev().assertElement(event.target),
       el => {
         tagName = el.tagName.toLowerCase();
-        if (tagName === 'amp-story-page-attachment') {
+        if (
+          tagName === 'amp-story-page-attachment' ||
```

Comment:
Should consider this separately, outside the scope of this pr, but it feels like we should invert this. Maybe components can declare a class like `i-amphtml-let-me-handle-the-clicking`, and the Page advancement just checks for the classname?

Reply:
Yeah I think that's a reasonable pattern to implement, we can spin that out into another PR